Add a file with a list of expectations for different roles (#10)
jenweber/emberjs-contributors-workshop: resources/expectations.md

# Expectations

This document outlines what is expected from the different roles involved in running the workshop, in terms of time and tasks.

## Hosts

1. Arranging mentors (5h)

You are responsible for arranging enough mentors to help contributors during the workshop.

> While a ratio of 1 mentor to 1 contributor is unrealistic, we recommend to keep it under 1 mentor to 5 contributors.

Reviewer: ```suggestion
While a ratio of 1 mentor to 1 contributor is unrealistic, we recommend to strive for at least 1 mentor to 5 every contributors.
```
This read a little strangely, so here's a suggested alternative.

1. Preparing the introduction material (1h)

Once you have hosts and mentors defined you should update the introductory slides to include that information.

1. Running the workshop (5h)

On the day of the actual workshop you should outline the workshop format and introduce the different mentors.
Then try to keep an eye on people having a hard time participating or finding work to do and try to help out.

> We recomend a 3h format, and that you leave yourself some prep and wind down time.

Reviewer: ```suggestion
We recommend a 3h format as well as leaving yourself some prep and wind down time.
```

> 1. Workshop follow-up (1h)

Reviewer: What does this entail?

Author: Added something!

## Mentors

> 1. Prepate issues for the workshop (3h) [BONUS]

Reviewer: ```suggestion
1. Prepare issues for the workshop (3h) [BONUS]
```
